### PR TITLE
[Release] Bump version to 1.0.0-alpha.7

### LINKED ISSUE #765: [Release] v1.0.0-alpha.7

### PR DESCRIPTION
**Pull request by feltech**
Closes #765

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -35,6 +35,42 @@ v1.0.0-alpha.x
   but with the additional openassetio dependencies installed into it.
```
Comment (foundrytom) [inline RELEASE_NOTES.md:35]: [fix]
```suggestion
  but with the additional OpenAssetIO dependencies installed into it.
```

**Review by foundrytom — APPROVED** [review]
LGTM - just a minor fix to squash in whilst you're here.

**Comment by foundrytom** [comment]
Should we do #761 first - if it facilitates the Conan work?

**Comment by feltech** [comment]
> Should we do #761 first - if it facilitates the Conan work?

After doing some research and experimentation, I think that is less of an issue now than it was. Maybe we should still do it, I'm not sure. See issue comments for details.

**Pull request merged**